_posts/2015-03-10-updated-site.markdown

```diff
@@ -0,0 +1,47 @@
+title:  "Site Updates and Changes"
+date:   2015-03-10 12:21:00
+categories: updates
+author: polymer-team
+excerpt: We're updating the Polymer site a bit as 0.8 is nearing release.
+
+---
+
+As many have already noticed, over the last day or so we've been updating the [Polymer site](https://www.polymer-project.org) to prepare for the upcoming 0.8 release. Some old links have broken in the process - we apologize for any troubles!
+
+The 0.8 release is a big step towards a 1.0: we're rolling out changes to the site to create a place to show 0.8 documentation, to enable better support for versioning, and generally to create a more stable home that will be able to grow with the library.
+
+# What to expect with the site for 0.8
+
+0.8 will be a new baseline for feedback - it will be hot out of the oven, but able to stand on its own. Consider it an "alpha" release - rough around the edges but generally whole, with the express purpose of getting feedback to roll into 1.0.
+
+With this in mind, we're working to accompany the release with a baseline of documentation to help early-adopters get started - expect the documentation to grow significantly as the features bake. The site itself for the time being will continue to be built based on 0.5 - we want to get 0.8 out for wide feedback as soon as possible, so having some documentation ready and starting to create 0.8-compatible elements has taken a priority over updating the site's library itself.
+
+We're also setting up the ability to version the documentation on the site - you'll notice that [https://www.polymer-project.org/](https://www.polymer-project.org/) currently takes you to https://www.polymer-project.org/0.5/ (the latest stable version). 0.8 will be transitional for the site as it is for the library and we'll be growing into this new versioning feature, but expect more and more firm structure like this to emerge.
+
+## What should you do now?
+
+### Update hotlinked URL's
+
+Many have found it helpful to be able to hotlink to the Polymer library hosted on the Polymer site, to lower develop-time overhead and to be able to easily create jsbin-like examples. With the new site versioning model, URL's will be of the form:
+
+`https://www.polymer-project.org/{version}/...`
+
+We certainly don't recommend relying on hotlinking for anything production. This new versioning should help hotlinking for development be more stable going forward, and help clarify any jsbin examples with an explicit version number.
+
+With regards to elements - we're working on major updates to the elements built by the Polymer team (you can see some progress in Github branches), including a better way of showcasing the element sets than we current have on the site. We'll keep the 0.5 elements the same as they are currently on the site. For 0.8, we plan on keeping compatible elements on Github for now as they're being built, and are working on new and improved ways to demo elements and generate and view their documentation.
```

Comment:
Should we mention there are already `0.8-preview` branches for some elements if people want to check them out? Or is that a no no since we're going to have a separate org?